modules/ui/app/scripts/services/RuleEditFactory.js

```diff
@@ -20,9 +19,7 @@ angular.module('wasabi.services').factory('RuleEditFactory', ['UtilitiesFactory'
                 { menuLabel: 'does not equal', stringForm: '!=', jsonForm: 'notEquals' },
                 { menuLabel: 'equals (same case)', stringForm: '^=', jsonForm: 'stringEqualsExact' },
```

Comment:
If you're OK with it, I think we'll just keep it the way it is, although I'll see if Calvin has an opinion.